modules/header/react/lib/parts/Search.tsx

```diff
@@ -298,40 +319,41 @@ export class Search extends React.Component<SearchProps, SearchState> {
   }
 
   render() {
-    const {onSearchSubmit, ...props} = this.props;
+    const {
+      themeColor,
+      placeholder,
+      value,
+      rightAlign,
+      collapse,
+      onSearchSubmit,
+      ...elementProps
+    } = this.props;
 
-    if (props.collapse) {
+    if (collapse) {
       return this._renderCollapsed();
     }
 
-    const iconColor =
-      this.state.hovered && this.state.focused && props.themeColor === HeaderTheme.White
-        ? colors.licorice500
-        : this.state.hovered && this.state.focused
-        ? colors.licorice500
-        : this.state.focused && props.themeColor === HeaderTheme.White
-        ? colors.licorice500
-        : this.state.focused
-        ? colors.licorice500
-        : this.state.hovered && props.themeColor === HeaderTheme.White
-        ? colors.licorice500
-        : this.state.hovered
-        ? colors.frenchVanilla100
-        : props.themeColor === HeaderTheme.White
-        ? colors.licorice200
-        : colors.frenchVanilla100;
-
     return (
-      <SearchContainer onSubmit={this.onSearchSubmit} {...props}>
-        <SystemIcon
+      <SearchContainer
+        rightAlign={rightAlign}
+        collapse={collapse}
+        onSubmit={this.onSearchSubmit}
+        {...elementProps}
+      >
+        <IconButton
+          aria-label="Search"
           icon={searchIcon}
-          style={{...iconStyle, pointerEvents: 'none'}}
-          color={iconColor}
-          colorHover={iconColor}
+          style={{...searchIconStyle}}
+          buttonSize={Button.Sizes.Small}
```

Comment:
What was the reason for using `Small` for these? This decreases the size of the icon so it no longer visually matches what we had before.

Reply:
the focus ring fit super weird on the non small version

Reply:
I just pushed a commit with some fixes after reviewing with Jesse. I basically overrode the container size so we could use 32px with a 24px icon. The only point of contention is that we decided to bump the text input height up 4px (but this is specific to the header). Let me know what you think.

Reply:
I think this is fine, don't think it will block moving search to a standard form component down the line. I also pushed a update to the reset button to match the plain styes and fix the case where there is a value but no focus on the blue header.